content/influxdb/v2.4/install.md

```diff
@@ -773,24 +834,24 @@ influx setup
    Enter nothing for an infinite retention period.
 7. Confirm the details for your primary user, organization, and bucket.
 
-InfluxDB is now initialized with a primary user, organization, bucket, and API token.
-InfluxDB also creates a configuration profile for you so that you don't have to
-add your InfluxDB host, organization, and token to every command.
-To view that config profile, use the [`influx config list`](/influxdb/v2.4/reference/cli/influx/config) command.
+Once setup completes, InfluxDB is initialized with the user, organization, bucket,
+and _[operator token](/influxdb/v2.4/security/tokens/#operator-token)_.
 
-To continue to use InfluxDB via the CLI, you need the API token created during setup.
-To view the token, log into the UI with the credentials created above.
-(For instructions, see [View tokens in the InfluxDB UI](/influxdb/v2.4/security/tokens/view-tokens/#view-tokens-in-the-influxdb-ui).)
+InfluxDB creates a `default` configuration profile for you that provides your
+InfluxDB URL, organization, and API token to `influx` CLI commands.
+For more detail about configuration profiles, see [`influx config`](/influxdb/v2.4/reference/cli/influx/config/).
 
-You are ready to [write or collect data](/influxdb/v2.4/write-data).
-
-{{% note %}}
-To automate the setup process, use [flags](/influxdb/v2.4/reference/cli/influx/setup/#flags)
-to provide the required information.
-{{% /note %}}
+Once you have the `default` configuration profile, you're ready to [create All-Access tokens](#create-all-access-tokens)
+or get started [collecting and writing data](/influxdb/v2.4/write-data).
 
 {{% /tab-content %}}
-<!------------------------------- END UI Setup -------------------------------->
+<!------------------------------- END CLI Setup -------------------------------->
 {{< /tabs-wrapper >}}
 
-After you’ve installed InfluxDB, you’re ready to [get started working with your data in InfluxDB](/influxdb/v2.4/get-started/).
+### Create All-Access tokens
+
+Because [operator tokens](/influxdb/v2.4/security/tokens/#operator-token)
```

Comment:
```suggestion
Because [Operator tokens](/influxdb/v2.4/security/tokens/#operator-token)
```

Reply:
> 

Why do we capitalize "All-Access token" and "Operator token", but not the names of other features?